content/en/docs/contributing/release-process.md

```diff
@@ -10,6 +10,61 @@ new release of cert-manager.
 
 ## Prerequisites
 
+<!--
+About "alert" and "pageinfo": https://www.docsy.dev/docs/adding-content/shortcodes/#shortcode-helpers
+-->
+
+{{% pageinfo color="warning" %}}
+
+⛔️ Do not proceed with the release process if you do not satisfy with one of the
+following:
+
+1. The release process **takes about 40 minutes**. You must have time to
+   complete all the steps.
+2. You need to be at Jetstack to create ask for the GitHub and GCP permissions.
```

Comment:
maybe change to:

> You currently need to be at Jetstack to get the required GitHub and GCP permissions.
> (we'd like contributors outside Jetstack to be able to get access; if that's of interest to you, please let us know)

Reply:
Thanks! Fixed in 2909939 👍

Reply:
Thanks! Fixed in da4b1e94c8380d3d9d9cdabf75f88aaa820564ba